trace-C19/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,92 @@
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #eeda73 !important;
+  text-align: center;
+  color: black;
+}
+
+.title {
+  color: black;
+  font-style: bold;
+  font-size: 400%;
+  text-align: center;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Lato", sans-serif;
+}
+
+.description {
+  font-size: 130%;
+  text-align: center;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Muli", sans-serif;
+  font-family: "Lato", sans-serif;
```

Comment:
Just curious as  to why we would need three font families ! Is is working as expected ?